lxc/image.go

```diff
@@ -881,7 +882,8 @@ func (c *imageCmd) run(conf *config.Config, args []string) error {
 				targetMeta = args[2]
 			}
 		}
-		targetRootfs := targetMeta + ".root"
+		targetMeta = shared.HostPath(targetMeta)
+		targetRootfs := shared.HostPath(targetMeta + ".root")
```

Comment:
That looks wrong, you've already prefixed targetMeta so you're double prefixing here